grunttasks/lint.js

```diff
@@ -8,6 +8,7 @@ module.exports = function (grunt) {
   'use strict';
 
   grunt.registerTask('lint', 'lint all the things', [
+    'amdcheck',
```

Comment:
I think this should probably be moved to [/grunttasks/concurrent.js:23-27](https://github.com/mozilla/fxa-content-server/blob/287179aaeffe1cdc7640745a93e54eedf2d975d3/grunttasks/concurrent.js#L23-L27).